libraries/botframework-connector/src/auth/botFrameworkClientImpl.ts

```diff
@@ -27,7 +27,15 @@ const botFrameworkClientFetchImpl: typeof fetch = async (input, init) => {
 };
 
 // Internal
+/**
+ * Implementation of BotFrameworkClient.
+ */
```

Comment:
We could change the `// Internal` comment, adding the `@internal` tag at the beginning of the JSDoc comment. Also add the xref for BotFrameworkClient